ABloom/ABloom/Presentation/CoreViews/QnA/View/AnswerWriteView.swift

```diff
@@ -35,7 +46,10 @@ struct AnswerWriteView: View {
           .foregroundStyle(.stone700)
       },
       leftView: {
-        Button(action: {dismiss()}, label: {
+        Button(action: {
+          answerVM.moveToBack()
+          
+        }, label: {
```

Comment:
```suggestion
        Button(action: answerVM.moveToBack, label: {
```

뷰 모델에 액션이 구현되어있다면, 이렇게 작성하는 것도 좋아요!

Reply:
오! 감사합니다 ㅎㅎ